sklearn/covariance/empirical_covariance_.py

```diff
@@ -69,7 +69,8 @@ def empirical_covariance(X, assume_centered=False):
     """
     X = np.asarray(X)
     if X.ndim == 1:
-        X = np.reshape(X, (1, -1))
+        X = X[:, np.newaxis]
```

Comment:
Well you changed the semantics of the code. Now the tests make more sense, but we should check when this changed. We can't just change behavior like this. Maybe someone relied on it being a single data point.